### PR TITLE
feat[web-16]: add responsive design for footer 

### PR DESCRIPTION
**Pull request by sofiiavoitkova**

### CODE REVIEW

File: src/components/Footer/Components/Logo/index.jsx [hunk 1]
```diff
@@ -5,8 +5,8 @@ import styles from './logo.module.scss'
 function Logo() {
   return (
     <div className={styles.logoContainer}>
-      <img src={logoTitle} alt="Logo Title" />
-      <img src={logoSubTitle} alt="Logo Subtitle" />
+      <img src={logoTitle} className={styles.logoTitle} alt="Logo Title" />
```
Comment (AndrZakhara) [inline src/components/Footer/Components/Logo/index.jsx]: use logo_title for className

File: src/components/Footer/Components/Logo/index.jsx [hunk 2]
```diff
@@ -5,8 +5,8 @@ import styles from './logo.module.scss'
 function Logo() {
   return (
     <div className={styles.logoContainer}>
```
Comment (AndrZakhara) [inline src/components/Footer/Components/Logo/index.jsx]: className={styles.container}

File: src/components/Footer/Components/Vlog/vlog.module.scss [hunk 3]
```diff
@@ -1,39 +1,55 @@
...
     letter-spacing: 0.03em;
+    width: 100%;
     font-family: var(--common-font-en), sans-serif;
+    
+    @media (max-width: $screen-tablet-l){
+        font-size: 1.25rem;
+    }
 }
 
 .vlogText {
```
Comment (AndrZakhara) [inline src/components/Footer/Components/Vlog/vlog.module.scss]: .text for example. don't use vlog

File: src/components/Footer/Components/Vlog/vlog.module.scss [hunk 4]
```diff
@@ -1,39 +1,55 @@
+@use '/src/styles/_variables' as *;
+
+.vlog{
+    width: 100%;
+}
+
 .vlogTitle {
```
Comment (AndrZakhara) [inline src/components/Footer/Components/Vlog/vlog.module.scss]: .title

File: src/components/Footer/Components/Vlog/vlog.module.scss [hunk 5]
```diff
@@ -1,39 +1,55 @@
+@use '/src/styles/_variables' as *;
+
+.vlog{
```
Comment (AndrZakhara) [inline src/components/Footer/Components/Vlog/vlog.module.scss]: .container

File: src/components/Footer/Components/Vlog/vlog.module.scss [hunk 6]
```diff
@@ -1,39 +1,55 @@
...
     color: var(--common-color-text-inverse-secondary);
     font-family: var(--common-font-en), sans-serif;
+
+    @media (max-width: $screen-tablet-l){
+        font-size: 0.9rem;
+        max-width: calc(14.5% + (900px - 60vw) * 0.3);
+    }
 }
 
 .vlogLink {
```
Comment (AndrZakhara) [inline src/components/Footer/Components/Vlog/vlog.module.scss]: .link

**Review by GarnaA — APPROVED** [review]

**Review by AndrZakhara — APPROVED** [review]

**Review by MyHKo — APPROVED** [review]

**Pull request merged**